### ISSUE TITLE
Migrate to sentry-sdk from raven

### ISSUE THREAD

**Issue by marco-c** [mozilla/code-review#999]
WIP patch:
```diff
diff --git a/tools/code_review_tools/log.py b/tools/code_review_tools/log.py
index fa51a93..3d4ba22 100644
--- a/tools/code_review_tools/log.py
+++ b/tools/code_review_tools/log.py
@@ -7,8 +7,8 @@ import os
 
 import logbook
 import logbook.more
-import raven
-import raven.handlers.logbook
+import sentry_sdk
+from sentry_sdk.integrations.logging import LoggingIntegration
 import structlog
 
 
@@ -58,23 +58,23 @@ def setup_sentry(name, channel, dsn):
     else:
         site = "unknown"
 
-    sentry_client = raven.Client(
+    sentry_logging = LoggingIntegration(
+        level=logging.INFO,          # Capture info and above as breadcrumbs
+        event_level=logging.WARNING  # Send warnings as events
+    )
+    sentry_sdk.init(
         dsn=dsn,
-        site=site,
-        name=name,
+        integrations=[sentry_logging],
+        server_name=name,
         environment=channel,
         release=raven.fetch_package_version(f"code-review-{name}"),
     )
+    sentry_sdk.set_tag("site", site)
 
     if task_id is not None:
         # Add a Taskcluster task id when available
         # It will be shown in the Additional Data section on the dashboard
-        sentry_client.context.merge({"extra": {"task_id": task_id}})
-
-    sentry_handler = raven.handlers.logbook.SentryHandler(
-        sentry_client, level=logbook.WARNING, bubble=True
-    )
-    sentry_handler.push_application()
+        sentry_sdk.set_context({"task_id": task_id})
 
 
 def init_logger(
```

We will need https://github.com/getsentry/sentry-python/issues/450 to finish this.

**Comment by marco-c** [comment]
> We will need getsentry/sentry-python#450 to finish this.

Or we can stop using Logbook.

**Comment by marco-c** [comment]
This was fixed by #1253.
